sys-kernel/dracut-crypt-ssh/dracut-crypt-ssh-1.0.7.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="early unlocking of encrypted systems via ssh for dracut"
```

Comment:
Can we get a capital `E` here in the beginning?

Reply:
Sure, done :)